front/pages/api/w/[wId]/assistant/conversations/[cId]/messages/index.ts

```diff
@@ -74,6 +81,47 @@ async function handler(
   }
 
   switch (req.method) {
+    case "GET":
+      const paginationRes = getKeysetPagination(req, {
```

Comment:
nit: would name getPaginationParams (took me a couple of cycles to get what it did here)